pages/requester/create-task.tsx

```diff
@@ -16,9 +16,25 @@ import { createTask } from "../../src/Database";
 import { useMoralis } from "react-moralis";
 import { useRouter } from "next/router";
 import { computeUnitRewardWei } from "../../src/Helpers";
+import { ethers, BigNumber } from "ethers";
+import EscrowFactory from "../../src/utils/abi/EscrowFactory.json";
+import Escrow from "../../src/utils/abi/Escrow.json";
+import ERC20ABI from "../../src/utils/abi/ERC20Token.json";
+import {
+  StackedLineChartSharp,
+  StackedLineChartTwoTone,
+} from "@mui/icons-material";
 
 export default function Form() {
-  const { isInitialized, Moralis } = useMoralis();
+  const escrowFactoryAddress: string =
+    process.env.NEXT_PUBLIC_ESCROW_FACTORY_ADDRESS;
+  const maticTokenAddress: string = process.env.NEXT_PUBLIC_MATIC_TOKEN_ADDRESS;
+  const { isInitialized, user, Moralis } = useMoralis();
+  // @ts-expect-error
+  const requesterAddress = user.get("ethAddress");
```

Comment:
Also, `@ts-expect-error` should be used rarely, if at all. For example, if there's a 100% unavoidable type error from an external library that has a bug or issue related to its types, then it might be ok.